application/apache-kafka-flink-streaming/.env

```diff
@@ -0,0 +1,15 @@
+CRATEDB_HOST=crate
+CRATEDB_PORT=4200
+CRATEDB_PG_PORT=5432
+
+WEATHER_PRODUCER_CITY=Viena
```

Comment:
Yeah, it's a typo, funny enough it didn't cause an issue because it actually exists;

```json
{
"location": {
        "name": "Viena",
        "region": "Quezon",
        "country": "Philippines",
        "lat": 13.98,
        "lon": 121.68,
        "tz_id": "Asia/Manila",
        "localtime_epoch": 1710256170,
        "localtime": "2024-03-12 23:09"
    }
 }
```